code/datums/wires/syndicatebomb.dm

```diff
@@ -14,48 +26,57 @@
 	if(P.open_panel)
 		return TRUE
 
+/datum/wires/syndicatebomb/repair()
+	. = ..()
+	delayedbig = FALSE
+	delayedlittle = FALSE
+	fake_delayed_little = FALSE
+	time_cut = FALSE
+
 /datum/wires/syndicatebomb/on_pulse(wire)
 	var/obj/machinery/syndicatebomb/B = holder
 	switch(wire)
-		if(WIRE_BOOM)
-			if(B.active)
-				holder.visible_message("<span class='danger'>[icon2html(B, viewers(holder))] An alarm sounds! It's go-</span>")
-				B.explode_now = TRUE
-				tell_admins(B)
-			else
+		if(WIRE_BOOM) // Only on cutting
+			if(fake_delayed_little)
 				holder.visible_message("<span class='notice'>[icon2html(B, viewers(holder))] Nothing happens.</span>")
+			else
+				holder.visible_message("<span class='notice'>[icon2html(B, viewers(holder))] The bomb seems to hesitate for a moment.</span>")
+				fake_delayed_little = TRUE
 		if(WIRE_UNBOLT)
 			holder.visible_message("<span class='notice'>[icon2html(B, viewers(holder))] The bolts spin in place for a moment.</span>")
 		if(WIRE_DELAY)
-			if(B.delayedbig)
+			if(delayedbig)
 				holder.visible_message("<span class='notice'>[icon2html(B, viewers(holder))] Nothing happens.</span>")
 			else
 				holder.visible_message("<span class='notice'>[icon2html(B, viewers(holder))] The bomb chirps.</span>")
 				playsound(B, 'sound/machines/chime.ogg', 30, 1)
-				B.detonation_timer += 300
+				B.detonation_timer += 10 SECONDS
 				if(B.active)
-					B.delayedbig = TRUE
+					delayedbig = TRUE
```

Comment:
The big delay is the same as the little delay, both are 10 seconds

Reply:
Changed the var names a bit to clarify